test/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/models/BusinessNameModelSpec.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.models
+
+import org.scalatest.Matchers._
+import org.scalatestplus.play.PlaySpec
+import org.scalatestplus.play.guice.GuiceOneServerPerSuite
+
+class BusinessNameModelSpec extends PlaySpec with GuiceOneServerPerSuite {
+
+  "BusinessNameModel" should {
+
+    val name = BusinessNameModel("Business")
+
+    "correctly format a business name" in {
+      name.businessName shouldBe "Business"
```

Comment:
Not required, just testing the functionality of scala at this point, can you add Json read / write tests?